addon/components/frost-table.js

```diff
@@ -60,6 +61,24 @@ export default Component.extend({
 
   // == Lifecycle Hooks =======================================================
 
+  didInsertElement () {
+    run.schedule('afterRender', this, function () {
+      const that = this
+      this.columns.forEach(function (column, index) {
+        const curBodyColumn = that.$().find('.frost-table-row .frost-table-body-cell:nth-child(' + (index + 1) + ')')
+        const curHeaderCell = that.$().find('.frost-table-header-cell:nth-child(' + (index + 1) + ')')
+
+        const bodyCellWidth = curBodyColumn.innerWidth()
+        const headerCellWidth = curHeaderCell.innerWidth()
+
+        let basis = bodyCellWidth > headerCellWidth ? bodyCellWidth : headerCellWidth
+
+        curHeaderCell.css('flex-basis', basis + 'px')
+        curBodyColumn.css('flex-basis', basis + 'px')
```

Comment:
Why modify the body columns?  I believe the intention was to have the header match the body, so the body shouldn't need modification